builtin/game/falling.lua

```diff
@@ -5,43 +5,43 @@
 --
 
 core.register_entity(":__builtin:falling_node", {
-	initial_properties = {
-		physical = true,
-		collide_with_objects = false,
-		collisionbox = {-0.5,-0.5,-0.5, 0.5,0.5,0.5},
-		visual = "wielditem",
-		textures = {},
-		visual_size = {x=0.667, y=0.667},
-	},
+	visual = "wielditem",
+	visual_size = {x = 0.667, y = 0.667},
+	textures = {},
+	physical = true,
+	is_visible = false,
```

Comment:
http://www.dict.cc/?s=deprecated

l think the deprecated refers to initial_properties because on_activate and on_step are nothing deprecated